sljit_src/sljitNativeARM_32.c

```diff
@@ -2712,14 +2710,16 @@ SLJIT_API_FUNC_ATTRIBUTE sljit_s32 sljit_emit_fset32(struct sljit_compiler *comp
 
 	u.value = value;
 
+#if defined(SLJIT_CONFIG_ARM_V7) && SLJIT_CONFIG_ARM_V7
 	if ((u.imm << (32 - 19)) == 0) {
-		exp = (u.imm >> (23 + 2)) & 0x3f;
+		sljit_u32 exp = (u.imm >> (23 + 2)) & 0x3f;
```

Comment:
This form is not used in the project. Add an ifdef around the definition.

Reply:
FWIW, locally scoped variables are valid C89